### ISSUE TITLE
Rich text editing - Guidance and code examples added to the design system site

### ISSUE THREAD

**Issue by halocline** [grommet/hpe-design-system#3288]
Parent issue: #3222 
Related issue: #3221 
Blocked by: https://github.com/grommet/hpe-design-system/issues/3332
- need usage guidance and code examples added to the design system site

**Comment by halocline** [comment]
Won't do. Complexity doesn't justify our efforts at this time. This feature was needed for an internal (i.e. not customer facing) use case.

Instead the recommendation is to author these posts in markdown and use the Markdown component.

If a true text editor is need, the current recommendation is to look at [QuillJS](https://github.com/slab/quill) because of active development, accessibility, and adoption by large companies like Salesforce and Slack.
